core/src/main/java/org/bptlab/cepta/operators/SumOfDelayAtStationFunction.java

```diff
@@ -52,7 +52,7 @@ public void process(Integer key, Context context, Iterable<TrainDelayNotificatio
                 HashMap<Long, Double> sums = new HashMap<Long, Double>();
                 for (TrainDelayNotification in: input) {
                     Long trainId = in.getTrainId();
-                    Long locationId = in.getLocationId();
+                    Long locationId = in.getStationId();
```

Comment:
is this locationId or stationId?  I think other parts of our code use stationId as the variablename but im not sure

Reply:
This change should solve all LocationID to StationID so there should be no LocationID anymore